docs/Post-Action-Registry.md

```diff
@@ -266,6 +267,24 @@ Adds `MyTestProject.csproj` to solution in output directory or its closest paren
 }]
 ```
 
+Adds `MyTestProject.csproj` in the root of the solution.
+
+```
+"primaryOutputs": [
+    {
+      "path": "MyTestProject.csproj"        
+    }
+],
+"postActions": [{
+  "description": "Add projects to solution",
+  "manualInstructions": [ { "text": "Add generated project to solution manually." } ],
+  "args": {
+    "inRoot": true
+  },
+  "actionId": "D396686C-DE0E-4DE6-906D-291CD29FC5DE",
+  "continueOnError": true
+}]
+```
```

Comment:
In a spirit of current effort to clean/improve our docs, please reformat the sample

````suggestion
```json
"primaryOutputs": [{
    "path": "MyTestProject.csproj"
  }
],
"postActions": [{
    "description": "Add projects to solution",
    "manualInstructions": [{
        "text": "Add generated project to solution manually."
      }
    ],
    "args": {
      "inRoot": true
    },
    "actionId": "D396686C-DE0E-4DE6-906D-291CD29FC5DE",
    "continueOnError": true
  }
]
```
````